packages/fluxible-router/lib/handleHistory.js

```diff
@@ -77,11 +74,6 @@ function createComponent(Component, opts) {
     Object.assign(HistoryHandler.prototype, {
 
         componentDidMount: function () {
-            if (historyCreated) {
-                throw new Error('Only one history handler should be on the ' +
-                'page at a time.');
-            }
-
```

Comment:
haha. guess we never set `historyCreated` to `true` as the logic intended.  i'm fine with removing it :)